### PR TITLE
Express integration for Transifex Native

### PR DESCRIPTION
**Pull request by kbairak**

### CODE REVIEW

**Comment by nbasili** [comment]
I would include `@transifex/native` as part of `@transifex/express` dependencies. This would simplify package management, make it part of the solution and instead of doing `tx.init({ .. })`, you could do just `const txExpress = new TxExpress({ token: XXX });`

Instead of being biased over `cookie`, `signed-cookie` and `session` modes , I would expect a way to be able to provide a function and an api to set the locale of the request, e.g. `new TxExpress({ locale: (req, res) => { ... }}` (unsure about the `locale` function name though).

Given the above, we could provide ready made strategies as part of the package, e.g. 
```
const { TxExpress, TxCookieLocale } = require('@transifex/express');

new TxExpress({ locale: TxCookieLocale });
```

In other words, offer an option for more control, and add recipes on top of it.

File: packages/cli/src/api/parsers/utils.js [hunk 1]
```diff
@@ -112,7 +112,18 @@ function isPayloadValid(payload, options = {}) {
...
+  if (callee.type === 'MemberExpression'
+    && callee.property.type === 'Identifier') {
+    fnName = callee.property.name;
+  } else if (callee.type !== 'Identifier') {
+    return false;
+  } else {
+    fnName = callee.name;
+  }
+
+  if (_.includes(['t', 'useT', '$t', 'translateLocale', 'ut'], fnName)) { return true; }
```
Comment (nbasili) [inline packages/cli/src/api/parsers/utils.js:126]: Not sure if `translateLocale` will work here, as this function takes the locale as a first argument. I think the code using this function expects the actual string to be on the first argument, so this will probably not work.

For example `translateLocale('en', 'Hello')` will make the CLI detect `en` as the source string.

Also `useT` should be removed from CLI, since this has changed when we made `useT` return a function. The code here was used when `useT` returned a translated string, e.g. `useT('Hello')`. We should also remove the related tests.

File: packages/express/README.md [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+Install the necessary express packages:
+
+```shell
+npm install --save express cookie-parser body-parser ...
+```
+
+And the Transifex Native integration:
+
+```shell
+npm install --save @transifex/express
```
Comment (nbasili) [inline packages/express/README.md:14]: Since you declare `@transifex/native` as a peerDependency, this will work only on NPM7. NPM prior to 7, does NOT automatically install peer dependencies, so this will not work. Two options:
- Add `@transifex/native` as dependency 
- Update the example here to `npm install --save @transifex/native @transifex/express`

File: packages/express/src/index.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+      for (let i = 0; i < tx.locales.length; i++) {
+        const locale = tx.locales[i];
+        /* eslint-disable no-await-in-loop */
+        await tx.fetchTranslations(locale);
+        /* eslint-enable */
+      }
+    };
+
+    /* eslint-disable no-console */
+    console.log('Transifex Native: fetching translations');
```
Comment (nbasili) [inline packages/express/src/index.js:160]: `console.log` should not be part of the code. I would remove it. If you need to have it, you could pass a `logging` param in the `TxExpress`, e.g `new TxExpress{{ logging: console.log })`. Sequelize library is doing this and it is works great.

File: packages/express/src/index.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    /* eslint-enable */
+    await _fetch();
+    /* eslint-disable no-console */
+    console.log('Transifex Native: done');
```
Comment (nbasili) [inline packages/express/src/index.js:164]: Same here

**Review by nbasili — CHANGES_REQUESTED** [review]
Added some single comments for review.

**Comment by kbairak** [comment]
All done. I also squashed. I believe it's ready to go. @nbasili thanks for your reviews. The comments were awesome!

**Review by nbasili — APPROVED** [review]
